rest_api/limsdb/queries/data_models.py

```diff
@@ -319,3 +320,69 @@ def to_json(self):
             'nb_reads': self.udfs['Read'],
             'nb_cycles': self.udfs['Cycle']
         }
+
+
+class Library:
+    def __init__(self):
+        self.id = None
+        self.type = None
+        self.qpcrs = defaultdict(QPCR)
+        self.projects = set()
```

Comment:
Can you store the projects in the `LibrarySample` object and aggregate it on the fly in `to_json`